deploy/iscsiadm-deps.yaml

```diff
@@ -0,0 +1,68 @@
+# Ubuntu 18.04 and above
```

Comment:
more comments would be good

Reply:
Please explain via comments that will be understood by a new user/contributor as well as to admin of the infra.